internal/servers/controller/handlers/hosts/host_service_test.go

```diff
@@ -253,8 +253,11 @@ func TestList_Static(t *testing.T) {
 	hcs := static.TestCatalogs(t, conn, proj.GetPublicId(), 2)
 	hc, hcNoHosts := hcs[0], hcs[1]
 
+	hset := static.TestSets(t, conn, hc.GetPublicId(), 1)[0]
 	var wantHs []*pb.Host
-	for _, h := range static.TestHosts(t, conn, hc.GetPublicId(), 10) {
+	testHosts := static.TestHosts(t, conn, hc.GetPublicId(), 10)
+	static.TestSetMembers(t, conn, hset.GetPublicId(), testHosts)
+	for _, h := range testHosts {
 		wantHs = append(wantHs, &pb.Host{
```

Comment:
Where is the SetIds expected on these wanted hosts?  I would expect either the test to fail if we got host set ids being returned by the API.  If they aren't failing then it seems like we are still not returning the set ids.